grassroots-backend/src/contacts/contacts.controller.ts

```diff
@@ -21,8 +22,14 @@ export class ContactsController {
     return this.contactsService.findAll();
   }
 
+  @ApiOkResponse({
+    description: "Returns a contact or null if not found",
+    schema: {
+      oneOf: [{ $ref: getSchemaPath(ContactEntityOutDTO) }, { type: "null" }],
+    },
+  })
   @Get(":id")
-  findOne(@Param("id") id: string): Promise<ContactEntityOutDTO | null> {
+  findOne(@Param("id") id: number): Promise<ContactEntityOutDTO | null> {
```

Comment:
Interesting that this doesn't work as expected

Reply:
I intentionally switched from string to number just to match the database schema. It was working fine, though there was an ugly cast which I've now gotten rid of.